include/deal.II/lac/vector.h

```diff
@@ -347,8 +347,8 @@ class Vector : public Subscriptor
 
 #ifdef DEAL_II_WITH_CXX11
   /**
-   * Move the given vector. This operator replaces the present vector with @p
-   * v by efficiently swapping the internal data structures.
+   * Move the given vector. This operator replaces the present vector with
+   * the internal data of the vector @p v.
```

Comment:
...and resets @p v to the state it would have been if newly default-constructed.